### PR TITLE
LRSD-4390 [PRM] Filter creation for Opp and Renewals

### PR DESCRIPTION
**Pull request by eliasantosf**
Task: [LRSD-4390](https://liferay.atlassian.net/browse/LRSD-4390) and [LRSD-4399](https://liferay.atlassian.net/browse/LRSD-4399)

### CODE REVIEW

**Comment by ryanschuhler** [comment]
ci:test:sf

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/PartnerOpportunitiesList/PartnerOpportunitiesList.tsx [hunk 1]
```diff
@@ -129,6 +142,70 @@ const PartnerOpportunitiesList = ({isRenewalListing, name}: IProps) => {
 		},
 	];
 
+	const todayDate = new Date();
+	const formattedDate = todayDate.toISOString().slice(0, 10);
```
Comment (ryanschuhler) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/PartnerOpportunitiesList/PartnerOpportunitiesList.tsx:146]: if order matters (which here it does, then we separate these two variables with a line break.

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/PartnerOpportunitiesList/utils/constants/getCloseDateFilterTerm.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export default function getCloseDateFilterTerm(
+	initialFilter: string,
+	activityPeriod: typeof INITIAL_FILTER.closeDate
+) {
+	const filterDates = [];
+
+	if (activityPeriod.dates.startDate) {
+		filterDates.push(`closeDate ge ${activityPeriod.dates.startDate}`);
+	}
+	if (activityPeriod.dates.endDate) {
```
Comment (ryanschuhler) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/PartnerOpportunitiesList/utils/constants/getCloseDateFilterTerm.ts:17]: there should be an extra line break to separate these two if blocks

**Review by ryanschuhler — CHANGES_REQUESTED** [review]
See comments, after those small tweaks I think it looks good to send to brian

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/PartnerOpportunitiesList/utils/constants/getCloseDateFilterTerm.ts (hunk 2 above)
Comment (eliasantosf) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/PartnerOpportunitiesList/utils/constants/getCloseDateFilterTerm.ts:18]: Done!

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/PartnerOpportunitiesList/PartnerOpportunitiesList.tsx (hunk 1 above)
Comment (eliasantosf) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/PartnerOpportunitiesList/PartnerOpportunitiesList.tsx:147]: Done!

**Review by ryanschuhler — APPROVED** [review]

**Comment by ryanschuhler** [comment]
ci:forward

**Pull request closed without merging**